singletons/utils.js

```diff
@@ -156,19 +156,75 @@ module.exports = (function (Module) {
 		 * Returns a formatted string, specifying an amount of time delta from current date to provided date.
 		 * @param {sb.Date|Date|number} target
 		 * @param {boolean} [skipAffixes] if true, the affixes "in X hours" or "X hours ago" will be omitted
+		 * @param {sb.Date} [deltaTo] If set, calculate time delta between target and deltaTo. If undefined, calculate
+		 * delta between target and the current time.
 		 * @returns {string}
 		 */
-		timeDelta (target, skipAffixes = false) {
-			const now = new sb.Date();
-			if (sb.Date.equals(now, target)) {
+		timeDelta (target, skipAffixes = false, deltaTo = undefined) {
```

Comment:
I added the deltaTo parameter to aid in unit testing. The mock for sb.Date is no longer needed after this.